src/app/contact/Contact.tsx

```diff
@@ -1,10 +1,74 @@
 'use client';
-import { Flex } from '@radix-ui/themes';
+import React, { type MouseEvent, useState } from 'react';
+import * as Label from '@radix-ui/react-label';
+import { Button, Flex, Heading } from '@radix-ui/themes';
+
+import styles from './contact.module.css';
+
+const Contact = () => {
+  const [formData, setFormData] = useState({
+    name: '',
+    email: '',
+    message: '',
+  });
+
+  const handleChange = (
+    e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    const { name, value } = e.target;
+    setFormData((prev) => ({ ...prev, [name]: value }));
+  };
+
+  const handleSubmit = (
+    e: MouseEvent | React.KeyboardEvent | React.FormEvent
+  ) => {
+    e.preventDefault();
+
+    //send email
+  };
 
-export default function Contact() {
   return (
-    <Flex direction={'column'} gap={'2rem 0'}>
-      <p>Contact page</p>
+    <Flex className={styles.container}>
+      <Heading>Contact Us</Heading>
+      <p>
+        We are here to assist you with any questions or feedback about Listify.
+      </p>
+      <form onSubmit={handleSubmit} className={styles.form}>
+        <Label.Root className="LabelRoot">Name:</Label.Root>
+        <input
+          className={styles.input}
+          type="text"
+          name="name"
+          value={formData.name}
+          onChange={handleChange}
+          required
+        />
+        <Label.Root className="LabelRoot">Email:</Label.Root>
+        <input
+          className={styles.input}
+          type="email"
+          name="email"
+          value={formData.email}
+          onChange={handleChange}
+          required
+        />
+
```

Comment:
Could we have these text fields with our styles?
For example, on create category
src/app/categories/components/AddEditCategory/styles.module.css

And by the way, should we use common style for login form?